local/languages/finna/en-gb.ini

```diff
@@ -454,7 +452,7 @@ Institutional Login = "Institutional Login of a University or a Polytechnic"
 Interior Images = "Interiors"
 Inventory ID = "Inventory ID"
 Item Description FWD = "Background"
-Item History = "Item History"
+Item History = "the history of the content/material"
```

Comment:
Onko tämä tosiaan pienellä alkukirjaimella? Suomeksi ja ruotsiksi isolla. Tarvitaanko alkuun "the"? Voisi olla vain "History...".

Reply:
Jep, muutetaan "History of the Content" tai "History of the Material". Käsittääkseni molempia termejä on ollut käytössä Finnassa.